### PR TITLE
cleanup FileLinks component unit test run warnings

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Cleanup `FileLinks` component unit test run warnings

## Related Issue

- Fixes https://github.com/owncloud/web/issues/6337

## How Has This Been Tested?

- :robot: 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/SideBar/Shares/FileLinks.spec.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import GetTextPlugin from 'vue-gettext'
 import DesignSystem from 'owncloud-design-system'
 import { mount, shallowMount, createLocalVue } from '@vue/test-utils'
 import FileLinks from '@files/src/components/SideBar/Shares/FileLinks.vue'
+import CreateForm from "../../../../../src/components/SideBar/Shares/Links/CreateForm";
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/FileLinks.spec.js:7]: Please use the `@files` import shorthand (see line above). Also, linter complains about the trailing `;` ;-)

Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/components/SideBar/Shares/FileLinks.spec.js]: :+1:

**Review by kulmann — APPROVED** [review]

**Comment by kiranparajuli589** [comment]
unit test run is now again nice and tidy https://drone.owncloud.com/owncloud/web/25265/10/7

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPublic1</strong> https://drone.owncloud.com/owncloud/web/25265/68/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Pull request merged**